cli/tests/integration_tests.rs

```diff
@@ -572,6 +572,21 @@ fn bundle_import_map() {
   assert_eq!(output.stderr, b"");
 }
 
+#[test]
+fn data_import() {
+  let status = util::deno_cmd()
+    .current_dir(util::tests_path())
+    .arg("test")
+    .arg("--unstable")
```

Comment:
Does it actually need --unstable? It seems like it would work without.

Reply:
Ah, a bad case of copy pasta